server/game/cards/07-WotW/ShinjoTrailblazer.js

```diff
@@ -0,0 +1,19 @@
+const DrawCard = require('../../drawcard.js');
+
+class ShinjoTrailblazer extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'Gain +2/+2',
+            when: {
+                onCardRevealed: (event, context) => event.card.isProvince && event.card.controller === context.source.controller.opponent && this.game.isDuringConflict()
```

Comment:
Please confirm if this is the best way to check if it's an opponent's province.  Seemed a bit wordy to me.

Reply:
I think that's the best way to do it....